urllib3/poolmanager.py

```diff
@@ -33,12 +34,14 @@
 # ``connection_pool_kw`` instance variable.
 HTTPPoolKey = collections.namedtuple(
     'HTTPPoolKey', BasePoolKey._fields + ('timeout', 'retries', 'strict',
-                                          'block', 'source_address')
+                                          'block', 'source_address',
+                                          'transport_security_manager')
 )
 HTTPSPoolKey = collections.namedtuple(
     'HTTPSPoolKey', HTTPPoolKey._fields + SSL_KEYWORDS
 )
 
+_DEFAULT_TSM = object()
 
 def _default_key_normalizer(key_class, request_context):
```

Comment:
Looks like the builder says you need to add an extra blank line here to make PEP8 happy.